### PR TITLE
chore: don't zip AppImages

### PR DESCRIPTION
**Pull request by jviotti**
See: https://github.com/AppImage/AppImageKit/wiki/Creating-AppImages#common-mistake
See: https://github.com/resin-io/etcher/pull/772
Signed-off-by: Juan Cruz Viotti <jv@jviotti.com>

### CODE REVIEW

**Comment by lurch** [comment]
I guess when this has been merged, and we do the next production release of Etcher, we'll need to remember to help @craig-mulligan update the filenames that he displays on etcher-homepage again ;)

**Comment by lurch** [comment]
I've tried restarting the Travis CI Linux builds for this a bunch of times, and they just keep failing in the middle of nowhere for no apparent reason :-/

I also had to restart the Appveyor build a few times before it passed - the first few times it failed because it couldn't install `curl`, and one time it failed with `ETIMEDOUT` when trying to run `scripts/html-lint.js`. Perhaps the whole internet is going wonky? ;)

**Comment by jviotti** [comment]
Looks like I missed a back slash. Lets see how it goes now.

**Comment by craig-mulligan** [comment]
@jviotti, @lurch please create an issue on the etcher website repo, if this does the filename listings.

File: README.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ Installers
 Refer to the [downloads page][etcher] for the latest pre-made
 installers for all supported operating systems.
 
+#### AppImage
+
+Set execution permissions on the `.AppImage` file and double-click it.
```
Comment (lurch) [inline README.md]: Should this say 'executable permissions' rather than 'execution permissions' ?

Comment (jviotti) [reply README.md]: See https://github.com/resin-io/etcher/pull/1715

**Pull request merged**